### PR TITLE
[Event Hubs]  Put uamqp.Message.properties into EventData.system_properties

### PR DESCRIPTION
**Pull request by YijunXieMS**
For issue #10408

### CODE REVIEW

**Comment by adxsdk6** [comment]
Can one of the admins verify this patch?

File: sdk/eventhub/azure-eventhub/azure/eventhub/_common.py [hunk 1]
```diff
@@ -207,12 +221,30 @@ def properties(self, value):
...
+                PROP_CORRELATION_ID: self.message.properties.correlation_id,
+                PROP_CONTENT_TYPE: self.message.properties.content_type,
+                PROP_CONTENT_ENCODING: self.message.properties.content_encoding,
+                PROP_ABSOLUTE_EXPIRY_TIME: self.message.properties.absolute_expiry_time,
+                PROP_CREATION_TIME: self.message.properties.creation_time,
+                PROP_GROUP_ID: self.message.properties.group_id,
+                PROP_GROUP_SEQUENCE: self.message.properties.group_sequence,
+                PROP_REPLY_TO_GROUP_ID: self.message.properties.reply_to_group_id,
+            }
+            self._sys_properties.update(self.message.annotations)
```
Comment (yunhaoling) [inline sdk/eventhub/azure-eventhub/azure/eventhub/_common.py]: why we're update the annotation here -- other languages are all having these information on "message.annotation"?

Comment (yunhaoling) [reply sdk/eventhub/azure-eventhub/azure/eventhub/_common.py]: I forget whether we support sending a received event out, but if we support that, then what will "message.properties" or "message.annotations" looks like if we receive that event again.

Comment (yunhaoling) [reply sdk/eventhub/azure-eventhub/azure/eventhub/_common.py]: oops, my bad, `a.update(b)` means `a` being updated.

File: sdk/eventhub/azure-eventhub/azure/eventhub/_common.py [hunk 2]
```diff
@@ -207,12 +221,30 @@ def properties(self, value):
 
     @property
     def system_properties(self):
-        # type: () -> Dict[Union[str, bytes], Any]
+        # type: () -> Dict[bytes, Any]
         """Metadata set by the Event Hubs Service associated with the event
 
         :rtype: dict
         """
```
Comment (yunhaoling) [inline sdk/eventhub/azure-eventhub/azure/eventhub/_common.py]: do we want to add comment here illustrating the keys we support?
like:
```
        Keys in the system properties dictionary include:
            - b`message-id` (bytes)
            - b`user-id` (bytes)
            - more ..
```

**Pull request merged**